### PR TITLE
[NTDDRAW][NTGDI][VGADDI] Add missing \n to DPRINT() calls

### PR DESCRIPTION
**Pull request by SergeGautherie**
## Purpose

Improve debug log.

## Proposed changes

- Add missing \n to DPRINT() calls
- and demote some DPRINT1().

### CODE REVIEW

File: win32ss/gdi/ntgdi/dibobj.c
Comment (JoachimHenze) [inline win32ss/gdi/ntgdi/dibobj.c:278]:
```suggestion
        DPRINT1("Compressed format without size\n");
```

File: win32ss/reactx/ntddraw/ddsurf.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/ddsurf.c:116]: I would recommend to not add pfn here, and in the few other places where you did that, but in this topic my opinion is not very strong.

File: win32ss/reactx/ntddraw/dvp.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/dvp.c:190]: We seem to have indentation with tabs here, can this be fixed?

File: win32ss/reactx/ntddraw/dvp.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/dvp.c:210]: tab indentation

File: win32ss/reactx/ntddraw/dvp.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/dvp.c:231]: tab indentation

File: win32ss/reactx/ntddraw/dvp.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/dvp.c:251]: tab indentation

File: win32ss/reactx/ntddraw/dvp.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/dvp.c:273]: tab indentation

File: win32ss/reactx/ntddraw/dvp.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/dvp.c:293]: tab indentation

File: win32ss/reactx/ntddraw/dvp.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/dvp.c:314]: tab indentation, and the neightbors don't have it. same as in the other cases. Should be fixable. And all is our code.

File: win32ss/reactx/ntddraw/dvp.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/dvp.c:334]: tab indent

File: win32ss/reactx/ntddraw/dvp.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/dvp.c:354]: tab indent. I will stop mentioning more of them now, even in case I will spot more.

File: win32ss/reactx/ntddraw/dvp.c
Comment (SergeGautherie) [reply win32ss/reactx/ntddraw/dvp.c:190]: Good catch. Fixed.

File: win32ss/reactx/ntddraw/ddsurf.c
Comment (SergeGautherie) [reply win32ss/reactx/ntddraw/ddsurf.c:116]: I added a few 'pfn' because it is almost everywhere else.
The other consistency option would be to remove it everywhere (in the strings).

File: win32ss/gdi/ntgdi/dibobj.c
Comment (SergeGautherie) [reply win32ss/gdi/ntgdi/dibobj.c:278]: Yeah, but not in this PR without author/maintainer approval.

File: win32ss/reactx/ntddraw/ddsurf.c
Comment (JoachimHenze) [reply win32ss/reactx/ntddraw/ddsurf.c:116]: Yes, most likely. I would recommend that, but it's okay to leave it for later if you would prefer that. As said: I am no extremist in this topic.

File: win32ss/gdi/ntgdi/dibobj.c
Comment (JoachimHenze) [reply win32ss/gdi/ntgdi/dibobj.c:278]: PROGRAMMER-field in header is empty. I do assume @DougLyons might have touched the file at some point.

File: win32ss/reactx/ntddraw/d3d.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/d3d.c:293]: The file has 4 space indentation. If you want to reformat that line, like you did, you should use 2x4, not 2x3.
```suggestion
        (PGD_DXDDDESTROYD3DBUFFER)gpDxFuncs[DXG_INDEX_DxDdDestroyD3DBuffer].pfn;
```

File: win32ss/reactx/ntddraw/d3d.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/d3d.c:319]: Same here, either do not touch it, or use 2x4, not 2x3.
```suggestion
        (PGD_D3DDRAWPRIMITIVES2)gpDxFuncs[DXG_INDEX_DxD3dDrawPrimitives2].pfn;
```

File: win32ss/reactx/ntddraw/d3d.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/d3d.c:359]: same here, either do not touch it, or use 2x4, not 2x3
```suggestion
        (PGD_D3DVALIDATETEXTURESTAGESTATE)gpDxFuncs[DXG_INDEX_DxD3dValidateTextureStageState].pfn;
```

File: win32ss/reactx/ntddraw/dvp.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/dvp.c:85]:
```suggestion
        (PGD_DVPDESTROYVIDEOPORT)gpDxFuncs[DXG_INDEX_DxDvpDestroyVideoPort].pfn;
```

File: win32ss/reactx/ntddraw/dvp.c
Comment (JoachimHenze) [inline win32ss/reactx/ntddraw/dvp.c:108]:
```suggestion
        (PGD_DVPFLIPVIDEOPORT)gpDxFuncs[DXG_INDEX_DxDvpFlipVideoPort].pfn;
```
I am not sure, how many will follow... if in doubt: I mean all of them.

File: win32ss/reactx/ntddraw/d3d.c
Comment (SergeGautherie) [reply win32ss/reactx/ntddraw/d3d.c:293]: Done, though, personally, I find it a bit confusing to use same indentation for a continued line as for a separate one.

**Review by HBelusca — APPROVED** [review]

File: win32ss/reactx/ntddraw/mocomp.c
Comment (oleg-dubinskiy) [inline win32ss/reactx/ntddraw/mocomp.c:67]: Make this function's alias same as others (without `Dx` prefix, since we already have `pfn`):
```suggestion
        DPRINT1("Warning: no pfnDdDestroyMoComp\n");
```

File: win32ss/reactx/ntddraw/mocomp.c
Comment (oleg-dubinskiy) [inline win32ss/reactx/ntddraw/mocomp.c:71]: Same comment here:
```suggestion
    DPRINT("Calling dxg.sys pfnDdDestroyMoComp\n");
```

File: win32ss/reactx/ntddraw/d3d.c
Comment (Doug-Lyons) [inline win32ss/reactx/ntddraw/d3d.c:71]: Probably "Calling on dxg.sys" could be changed to just "Calling dxg.sys" on this and the next 14 cases, but not a big deal. I will approve anyway. Thanks.

**Review by Doug-Lyons — APPROVED** [review]

**Review by oleg-dubinskiy — APPROVED** [review]
In other than that LGTM. :slightly_smiling_face:

**Review by JoachimHenze — APPROVED** [review]

**Comment by HBelusca** [comment]
Going to wait for @SergeGautherie to add the latest suggestions to this PR before merging it.

**Review by SergeGautherie — COMMENTED** [review]
Plus, I added 5 other NTDDRAW files, which did not have the main issue.

**Pull request merged**
